README.md

```diff
@@ -52,8 +52,7 @@ deploy and configure the platform.
 
 ## Acknowledgement
 
-This toolkit was inspired by the roles and recipes that Ángel Jara developed
-when he worked at Bitergia and by the amazing job the people of
+This toolkit was inspired by the roles and recipes that [Ángel Jara](https://github.com/ajaragz) developed when he worked at Bitergia and by the amazing job the people of
```

Comment:
Please limit the length of this line to 80 characters.

Reply:
done